viewer/src/main/webapp/viewer-html/components/Edit.js

```diff
@@ -1005,7 +1001,19 @@ Ext.define("viewer.components.Edit", {
             editable: !(attribute.hasOwnProperty('allowValueListOnly') && attribute.allowValueListOnly),
             labelClsExtra: this.editLblClass
         });
-
+        
+        //when device is mobile dont allow the touchend event. This will activate a field that is lying under the dropdown picker(ul)
+        if (viewer.components.MobileManager.isMobile()) {
```

Comment:
Is this only an issue here or should we move this fix to overrides and fix all comboboxes?